src/pluto/reader/hooks.cljc

```diff
@@ -20,11 +23,24 @@
 (defmethod resolve-property :view [opts m def hook]
   (views/parse opts (get m (reference->symbol (property-value def hook)))))
 
+(defn resolve-property-value [f {:keys [name] :as def} hook]
+  (if-let [o (property-value def hook)]
+    (if (f o)
```

Comment:
`(f o)` ? O_o

Reply:
what is `f` what is`o` ?

Reply:
I use `f` for function and `o` for object (like anything). But yeah could be more explicit!